cmd/monaco/main.go

```diff
@@ -212,7 +213,7 @@ Examples:
 
 	return app
 }
-func getDeployCommand(fileReader util.FileReader) cli.Command {
+func getDeployCommand(fileReader afero.IOFS) cli.Command {
```

Comment:
Nitpick: please rename all variables of type `afero.IOFS` to `fs`. 
`fileReader` isn't really the right name anymore.